packages/sources/amberdata/src/config.ts

```diff
@@ -21,3 +24,41 @@ export const makeConfig = (prefix = ''): Config => {
   config.api.baseURL = config.api.baseURL || DEFAULT_API_ENDPOINT
   return config
 }
+
+export const makeWSHandler = (defaultConfig?: Config): MakeWSHandler => {
+  const subscriptions: any = {}
+  const getPair = (input: AdapterRequest): string => {
+    const validator = new Validator(input, endpoint.price.customParams)
+    if (validator.error) {
+      return ''
```

Comment:
We should not swallow an error like this. If we can not construct a subscribe/unsubscribe message, we should return `undefined` that the WS component should handle.